src/Backend/Modules/MediaLibrary/Domain/MediaItem/Type.php

```diff
@@ -87,6 +89,8 @@ public static function fromExtension(string $extension): self
                 return self::fromString($mediaItemType);
             }
         }
+
+        throw new Exception('Extension could not be resolved');
```

Comment:
Maybe use a less generic exception?

Reply:
Used InvalidArgument since it is about a string and not a specific class